woocommerce/woocommerce
# Change the logo and message for WooCommerce tax during onboarding

### Changes proposed in this Pull Request:

This pull request is a follow-up of https://github.com/woocommerce/woocommerce/pull/53308, it changes the logo of WooCommerce Tax that appears in the "Collect sales tax" onboarding task.

![image](https://github.com/user-attachments/assets/3a30a2ef-5ecf-4a32-aa1d-42bf13c79200)

### How to test the changes in this Pull Request:

1. Activate the new branding as instructed in https://github.com/woocommerce/woocommerce/pull/53308

2. Go to the WooCommerce Dashboard (WooCommerce - Home), then in the "Start customizing your store" box click "Collect sales tax" and see how the new WooCommerce icon shows. If you can't see the box you may need to reset the onboarding via the "Help" menu that appears in the top right corner of the WooCommerce - Status page (Setup Wizard - Task List - Enable).

3. Deactivate the new branding, reload the page, and verify that you see the old logo again.

### Changelog entry












<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
